src/flutter/shell/platform/linux_embedded/surface/environment_egl.h

```diff
@@ -28,8 +28,8 @@ class EnvironmentEgl {
 
   EnvironmentEgl() : display_(EGL_NO_DISPLAY), valid_(false) {}
```

Comment:
sub_environment_(false) is missing. Is this intentional?

Reply:
Ultimately I deleted EnvironmentEglSub class and refactored source files.